ipykernel/kernelbase.py

```diff
@@ -859,6 +860,22 @@ async def debug_request(self, stream, ident, parent):
                                       parent, ident)
         self.log.debug("%s", reply_msg)
 
+    async def usage_request(self, stream, ident, parent):
+        content = parent['content']
+        reply_content = self.do_debug_request(content)
```

Comment:
why calling this method specific do `debug_request` here?

Reply:
Not needed. Latest commit removes it. Thx @JohanMabille